### PR TITLE
updated calls to python -> python3. Added reminder file generation

### PR DESCRIPTION
**Pull request by RafaelPiloto10**
Changed most calls in code to `python` to now use `python3` as per issue #5.
Also added feature to auto generate reminders file if it does not exist so that it does not have to be a part of the repository.

### CODE REVIEW

File: src/GIS_main.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def run(inputFilename,
 			if inputFilename.endswith('.csv'):
 				configArray=['EMPTY LINE', inputFilename, 'EMPTY LINE', 'EMPTY LINE', 'EMPTY LINE', outputDir]
 				config_util.saveConfig(GUI_util.window, 'GIS-Google-Earth-config.txt', configArray,True)
-			call("python GIS_Google_Earth_main.py", shell=True)
+			call("python3GIS_Google_Earth_main.py", shell=True)
```
Comment (DDVD233) [inline src/GIS_main.py]: Is a space missing here?

File: src/GUI_util.py [hunk 2]
```diff
@@ -363,7 +363,7 @@ def activateRunButton(*args):
 
     if (os.path.isfile(os.path.join(GUI_IO_util.libPath, 'LICENSE-NLP-1.0.txt'))) and (IO_libraries_util.inputProgramFileCheck('license_GUI.py')):
         if not os.path.isfile(GUI_IO_util.configPath + os.sep + 'license-config.txt'):
-            call("python " + "license_GUI.py", shell=True)
+            call("python3" + "license_GUI.py", shell=True)
```
Comment (DDVD233) [inline src/GUI_util.py]: Also here

**Review by DDVD233 — CHANGES_REQUESTED** [review]
It looks like some of the spaces after python3 are missing. Could you fix that?

**Comment by Emory-NLP** [comment]
I have not pulled Rafael's code. So I do not have that error. I'll wait for Rafael before I end up losing some work or screwing up :-).

**Comment by RafaelPiloto10** [comment]
Good catch @zjdavid, I have made the changes as well as updated a couple of the install scripts to also use `python3`. This merge would also address #31. The reminders file that is generated is not yet in the default configuration. @Emory-NLP, what should a default reminders.csv file look like? (all yes? all no? etc). After that last change, we can merge this request.

**Comment by RafaelPiloto10** [comment]
Further testing revealed that the reminders.csv file was being created incorrectly. Changed method of generating the file using a pre-existing default csv file. Should now properly address #31

**Pull request merged**